### DISCUSSION TITLE
Remove Upload limit of web interface

### DISCUSSION THREAD

**Discussion by maxomatic458** [pelican-dev/panel#487] [Ideas]
The max value for the upload is 1024 MB, but i think it should be possible to remove this limit (as an admin).

especially if you run pterodactyl in a local network, its just more convinient to upload files via the web interface

**Comment by QuintenQVD0** [comment]
This won't happen. Web servers are not made for big file uploads. There is SFTP for a reason

**Comment by notAreYouScared** [comment]
This will not happen, if you have a file bigger than 1GB you should really be using the SFTP Server built into wings.
